packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -314,7 +317,10 @@ export class DocumentDeltaConnection
 	}
 
 	protected emitMessages(type: "submitOp", messages: IDocumentMessage[][]): void;
-	protected emitMessages(type: "submitSignal", messages: string[][]): void;
+	protected emitMessages(
+		type: "submitSignal",
+		messages: string[][] | ISentSignalMessage[],
+	): void;
```

Comment:
This is an override. Is it possible to add a comment about `ISentSignalMessage[]` only being okay when service supports feature ... and that comment appears at the right time?
(Maybe you need to add a comment that covers all of the overrides. I think that is how it works, but haven't played.)